src/triggers.js

```diff
@@ -32,7 +33,8 @@ const baseStore = function() {
 };
 
 function validateClassNameForTriggers(className, type) {
-  const restrictedClassNames = ['_Session'];
+  // Contains classes which do not allow triggers of any kind
+  const restrictedClassNames = [];
   if (restrictedClassNames.indexOf(className) != -1) {
```

Comment:
This can be deleted no?

Reply:
Yes, I thought I leave it as placeholder, but who knows if we ever need it again. Will remove it.